mmv1/products/colab/NotebookExecution.yaml

```diff
@@ -144,6 +154,75 @@ properties:
     description: 'The NotebookRuntimeTemplate to source compute configuration from.'
     exactly_one_of:
       - notebook_runtime_template_resource_name
+      - custom_environment_spec
+  - name: customEnvironmentSpec
+    type: NestedObject
+    description: Compute configuration to use for an execution job
+    exactly_one_of:
+      - notebook_runtime_template_resource_name
+      - custom_environment_spec
+    properties:
+      - name: machineSpec
+        type: NestedObject
+        default_from_api: true
```

Comment:
is it necessary for all these fields to be default_from_api? we would have no way to unset any fields.